packages/cactus-test-plugin-ledger-connector-quorum/src/test/typescript/integration/api-surface.test.ts~HEAD

```diff
@@ -0,0 +1,9 @@
+const tap = require("tap");
```

Comment:
@jagpreetsinghsasan Please consolidate the test files, these file extensions are off and there's also a rename that drops the .test.ts suffix for one of them